inst/using-the-dccvalidator-pec.Rmd

```diff
@@ -109,7 +109,7 @@ The [Data Analysis Core](https://www.synapse.org/#!Synapse:syn20959298) will rep
   
 ## Uploading data to Synapse after validation
 
-Once data has passed validation and the PEC data curators permit edit permissions to the **Staging** folder, data may be uploaded. Bulk file upload is achievable using the [web UI](https://docs.synapse.org/articles/files_and_versioning.html) or the [R, Python and command line clients](https://docs.synapse.org/articles/uploading_in_bulk.html).
+Once data has passed validation, and the PEC data curators permit edit permissions to the **Staging** folder, you will use your newly created manifest file to upload data using `syncToSynapse`. You can execute `synToSynapse` in the [Python client](https://python-docs.synapse.org/build/html/synapseutils.html#synapseutils.sync.syncToSynapse) and [R client](https://github.com/Sage-Bionetworks/synapserutils#upload-data-in-bulk). For getting started with the Synapse programmatic clients, please visit our [Synapse docs](https://docs.synapse.org/articles/api_documentation.html).
```

Comment:
```suggestion
Once data has passed validation, and the PEC data curators permit edit permissions to the **Staging** folder, you will use your newly created manifest file to upload data using `syncToSynapse`. You can execute `syncToSynapse` in the [Python client](https://python-docs.synapse.org/build/html/synapseutils.html#synapseutils.sync.syncToSynapse) and [R client](https://github.com/Sage-Bionetworks/synapserutils#upload-data-in-bulk). For getting started with the Synapse programmatic clients, please visit our [Synapse docs](https://docs.synapse.org/articles/api_documentation.html).
```

Reply:
@karawoo I don't see a difference in your suggested change! Am I missing something?

Reply:
You had spelled `syncToSynapse` as `synToSynapse` in one spot
<img width="129" alt="Screen Shot 2020-02-06 at 9 55 46 AM" src="https://user-images.githubusercontent.com/4452678/73964380-e8855c80-48c6-11ea-94dc-cdd1e851d424.png">